netbench-cdk/Makefile

```diff
@@ -18,7 +18,7 @@ endif
 
 deploy: build
 ifdef DEV_ACCOUNT_ID
-	cdk deploy NetbenchInfraDev-$(USER) --require-approval never
+	cdk deploy NetbenchInfraDev-$(USER) --require-approval never -O output.json
```

Comment:
can we name this `cdk_config.json` since the orchestrator will have quite a few params